Multisig/UI/ClaimToken/Select Guardian/GuardianDetailsViewController.swift

```diff
@@ -59,6 +60,36 @@ class GuardianDetailsViewController: UIViewController {
         } else {
             viewOnEtherscan.url = chain.browserURL(address: guardian.address.address.checksummed)
         }
+
+        let reasonTapRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap(sender:)))
```

Comment:
Why do we need this feature ?

Reply:
because one guardian posted a youtube link in contributions and there's no way to open it.

I added copy text for everything else